Vagrantfile

```diff
@@ -1,6 +1,7 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
-VAGRANTFILE_API_VERSION = '2' unless defined? VAGRANTFILE_API_VERSION
```

Comment:
I don't really see the point of defining this when it could technically be overridden by an earlier `vagrant` command.